src/Introspection/__EnumValue.hack

```diff
@@ -0,0 +1,19 @@
+namespace Slack\GraphQL\Introspection;
+
+use namespace Slack\GraphQL;
+
+<<GraphQL\InterfaceType('__EnumValue', 'Enum value introspection')>>
+interface __EnumValue {
```

Comment:
Some of these could just be classes instead of interfaces, e.g. I don't think we'll need a non-introspection version of EnumValue that would implement this interface.

Reply:
Yea that makes sense. I haven't implemented anything with enums yet and will do that in a follow up PR

Reply:
Some of these could even be shapes maybe, now that we support generating object types from shapes.

```
<<GraphQL\Object('__EnumValue')>>
type __EnumValue = shape(
  'name' => string,
  // ...
);
```